src/Sulu/Bundle/AdminBundle/Resources/js/containers/FieldBlocks/FieldBlocks.js

```diff
@@ -121,6 +121,38 @@ class FieldBlocks extends React.Component<FieldTypeProps<Array<BlockEntry>>> {
         return addButtonText;
     }
 
+    @computed get collapsable() {
```

Comment:
we need to document this

Reply:
Done in https://github.com/sulu/sulu-docs/pull/614